diff --git a/node_modules/hono-fsr/dist/lib/utils.js b/node_modules/hono-fsr/dist/lib/utils.js
index 99e1e9f..bb5fec9 100644
--- a/node_modules/hono-fsr/dist/lib/utils.js
+++ b/node_modules/hono-fsr/dist/lib/utils.js
@@ -1,5 +1,5 @@
 import path from "node:path";
-const VALID_EXTENSIONS = [".ts", ".mts", ".js", ".mjs", ".cjs"];
+const VALID_EXTENSIONS = [".ts", ".tsx", ".mts", ".js", ".mjs", ".cjs"];
 function calculatePrecedence(urlPath) {
     if (urlPath.includes("*"))
         return 3;
